kolibri/utils/debian_check.py

```diff
@@ -0,0 +1,29 @@
+import os
+import sys
+
+from builtins import input
+
+
+def check_debian_user():
+    # Check whether the current user is the kolibri user when running kolibri
+    # that is installed from .deb package.
+    # The code is mainly from https://github.com/learningequality/ka-lite/blob/master/bin/kalite#L53
+    if os.name == "posix" and os.path.isfile("/etc/kolibri/username"):
+        kolibri_user = open("/etc/kolibri/username", "r").read().rstrip()
+        current_user = os.environ["USER"]
+        if kolibri_user and kolibri_user != current_user:
+            kolibri_home = os.environ.get("KOLIBRI_HOME", os.path.expanduser("~/.kolibri"))
+            if not os.path.exists(kolibri_home) or not os.listdir(kolibri_home):
+                sys.stderr.write((
+                    "You are not running kolibri as the default user {}. "
+                    "This is recommended unless you want to re-create the database "
```

Comment:
Shouldn't this be `this is NOT recommended`, based on the tone of the rest of the sentence?

Reply:
yeah, this sentence is a bit confusing. Let me work on the copy

Reply:
what do you two think of this?


> You are running this command as the user '{current_user}', but Kolibri was originally installed  to run as the user '{kolibri_user}'. This may result in unexpected behavior, because the two users will each use their own local databases and content.
> 
> If you'd like to run the command as '{kolibri_user}', you can type 'n' below and try:
> 
>      sudo su {kolibri_user} -c '<command>'
> 
> Alternatively, would you like to continue and running the command as '{current_user}'? [y/n]

Reply:
> you can type 'n' below

Actually in the current logic, to exit the prompt, users can type anything except 'y' or 'Y' or just press the enter key. This is to follow some convention in Debian. So I'm wondering if we can just say:
> If you'd like to run the command as '{kolibri_user}', you can try:

Reply:
okay that's fine with me

Reply:
thanks! I'm going to make the changes now

Reply:
![screen shot 2018-06-29 at 3 10 18 pm](https://user-images.githubusercontent.com/19424916/42116917-9bc58a8c-7bae-11e8-96a4-642d9592747c.png)
@aronasorman @indirectlylit Here is what the new prompt looks like. Please let me know if I need to make more changes. Thank you! :)

Reply:
For the last sentence, should it be `continue running` or `continue and run`? Thanks!

Reply:
Good catch! 'continue and run' please